infra/src/main/java/com/redhat/service/smartevents/infra/api/APIConstants.java

```diff
@@ -113,9 +113,9 @@ public class APIConstants {
     public static final String SIZE_DEFAULT = "100";
 
     /**
-     * The name query filter parameter name
+     * The name query filter parameter prefix
      */
-    public static final String FILTER_NAME = "name";
+    public static final String PREFIX = "namePrefix";
```

Comment:
IMO `FILTER_NAME_PREFIX` would have been nicer as it relates only to the `Name` field.

Making the other parameters `filterNamePrefix` etc.

Reply:
done